.github/workflows/test-using-pytest.yml

```diff
@@ -50,20 +50,26 @@ jobs:
           cp /tmp/firecracker-release/release-v*/jailer-v*[!.debug] /opt/firecracker/jailer
           chmod +x /opt/firecracker/firecracker
           chmod +x /opt/firecracker/jailer
-          
+
           find /opt
 
-      - name: "Build custom runtime"
+      - name: "Build custom runtimes"
         run: |
           sudo apt update
           sudo apt install -y debootstrap ndppd acl cloud-image-utils qemu-utils qemu-system-x86
           cd runtimes/aleph-debian-12-python && sudo ./create_disk_image.sh && cd ../..
+          cd runtimes/instance-rootfs && sudo ./create-ubuntu-22-04-qemu-disk.sh && cd ../..
+          cd runtimes/instance-rootfs && sudo ./create-debian-12-disk.sh && cd ../..
 
       - name: "Build example volume"
         run: |
           cd examples/volumes && bash build_squashfs.sh
 
-        # Unit tests create and delete network interfaces, and therefore require to run as root
+      - name: Create user jailman
+        run: |
+          sudo useradd jailman
```

Comment:
I think that this line can be added at final in a previous step, like in the `Download and build required files for running tests. Copied from packaging/Makefile.` that install jailer.